spec/controllers/admin/payments_controller_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+
+describe Admin::PaymentsController do
+  describe "GET index" do
+    it_behaves_like 'require_admin' do
+      let(:action) { get :index }
+    end
+
+    it "shows the last 10 records of payment" do
```

Comment:
Oh yes, the description should be more precise about what tests are validating. I add expectation to validate that the variable is the last ten records.